### DISCUSSION TITLE
Temperature in Kelvin?

### DISCUSSION THREAD

**Discussion by quantenzitrone** [polybar/polybar#2774] [Ideas]
Since the temperature module has support for temperature in °C and °F, I thought it would be nice to support K as well, since Kelvin is the standard SI-Unit for temperature. 
I looked over the temperature.cpp file, and it seems to me, as if adding the temperature in Kelvin would be as easy as adding a few lines to define Kelvin as a formatting option and calculating Kelvin from °C with `int temp_k = floor(m_temp + 273.15);`

**Comment by patrick96** [comment]
That's seems easy enough to add. Do you personally have a use-case for this? I am not inclined to accept new features if no one actually wants them

**Reply by quantenzitrone** [reply]
I would use it, but other than me probably not a lot of people, maybe a few nerds that believe in Kelvin supremacy, like me.

**Reply by patrick96** [reply]
If you're going to use it, feel free to open a PR ;)
